### ISSUE TITLE
Android Studio: Cannot detect installed Kotlin Multiplatform Mobile Plugin

### ISSUE THREAD

**Issue by wumo** [Kotlin/kdoctor#25]
Running `kdoctor` shows:
```
[x] Android Studio
    * Android Studio (2021.3)
      Location: /Users/wumo/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/213.7172.25.2113.9014738/Android Studio.app
      Bundled Java: openjdk 11.0.13 2021-10-19
      Kotlin Plugin: 213-1.7.10-release-for-android-studio-AS6777.52
      Kotlin Multiplatform Mobile Plugin: not installed
          Install Kotlin Multiplatform Mobile plugin - https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile
```
But the plugin is installed:
![WeChatWorkScreenshot_7919f94a-38d4-4240-a4b2-9fc3a88af95a](https://user-images.githubusercontent.com/7351935/191481396-69926aee-597a-421f-841e-27b0285d3f38.png)

**Comment by wblondel** [comment]
Same issue for me.

Partial `kdoctor` output:
    
```
[x] Android Studio
    * Android Studio (2021.3)
      Location: /Users/wblondel/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/213.7172.25.2113.9014738/Android Studio.app
      Bundled Java: openjdk 11.0.13 2021-10-19
      Kotlin Plugin: 213-1.7.10-release-for-android-studio-AS6777.52
      Kotlin Multiplatform Mobile Plugin: not installed
          Install Kotlin Multiplatform Mobile plugin - https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile
```

Installed plugin:
<img width="1198" alt="image" src="https://user-images.githubusercontent.com/7508531/193443427-d9f2ce23-5990-472a-9a92-7139ffa18cab.png">


macOS 12.6 on M1

**Comment by findjigar** [comment]
@wumo @wblondel 
Maybe you've multiple Android Studio installed, and the one you're running is not the same one mentioned in the error you're getting?

**Comment by wumo** [comment]
@findjigar 
No, only one version android studio is installed.
I also check the android studio app listed in kdoctor and kotlin multiplatform mobile plugin is installed.

**Comment by mhorioka** [comment]
Hi, I have the same issue and I believe kdoctor does not check the location of kmm plugin correctly.

As I tested, kdocter looks for plugins installed in a default location like:

- $HOME/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-1/222.4345.14.2221.9178080/Android Studio Preview.app/Contents
- $HOME/Library/Application Support/Google/AndroidStudioPreview2022.2

but, with JetBrains Toolbox App, the kmm plugin was actually installed in the "$location.plugins" directory like:

$HOME/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-1/222.4345.14.2221.9178080/Android Studio Preview.app.plugins

So I guess https://github.com/mhorioka/kdoctor/blob/3d15c6f34790fa85097bad35108d32759fc7f223/kdoctor/src/macosMain/kotlin/org/jetbrains/kotlin/doctor/entity/Application.kt needs to be updated.

**Comment by jQrgen** [comment]
I have the same issue and I have installed Android Studio trough Jetbrains Toolbox

**Comment by Sartoric** [comment]
Yep, same for me.
The issue is still there, someone should review the PR :)

**Comment by terrakok** [comment]
I've fixed the issue during refactoring `kdoctor`. Should be OK in the next release

**Comment by terrakok** [comment]
I've fixed the issue during refactoring `kdoctor`. Should be OK in the next release

If you want, you can try to run it via `./gradlew runDebugExecutableMacosX64` on `wip` branch

**Comment by sulozx** [comment]
The easiest way to make it work is to create a soft link:
```
cd $HOME/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/213.7172.25.2113.9123335/Android Studio.app/Contents/plugins 
ln -s ../../../Android\ Studio.app.plugins/kmm .
````

**Comment by terrakok** [comment]
it should be fixed in 0.0.5 version
